test/Table_Tests/src/Common_Table_Operations/Date_Time_Spec.enso

```diff
@@ -522,12 +522,12 @@ add_specs suite_builder setup =
 
     if setup.test_selection.date_time.not then
         suite_builder.group prefix+"partial Date-Time support" group_builder->
-            group_builder.specify "will fail when uploading a Table containing Dates" <|
+            group_builder.specify "will fail when uploading a Table containing Dates" pending="TODO" <|
```

Comment:
how come we need to make this as pending?
Different error now?

Reply:
Yeah SQLServer is currently in a halfway house on support. Will add full support soon and then can re-enable this test